### ISSUE TITLE
[WISH] Place to put thanks and greatings

### ISSUE THREAD

**Issue by zeroheure** [trunkmaster/nextspace#245]
This not a real issue. But a place for everyone to put thanks and greatings to Sergii Stoian who is working for a very long time on this project. :-)

**Comment by zeroheure** [comment]
I check your progress very often, and I must say that every time I smile because of the impressive and frequent changes. I lack some time to help (other personal projects...) but thankyou for all your work.

**Comment by trunkmaster** [comment]
@zeroheure, thank you very much for kind words! I'm glad you like what I'm doing.
You've read my last post at https://trunkmaster.github.io, right? ;-)

Your unexpected message led me to some ideas:
1. This issue may be a place where I can put some hot news in Twitter like manner.  Sometimes I lack inspiration/time to write long posts on https://trunkmaster.github.io. Anyone interested can subscribe to this issue to be in touch with latest news and progress.
2. I guess it would be great some web page with project description and other materials related to the project. I'm do not have expertise in HTML, JavaScript and so on. Although GitHub has a place for it - GitHub Pages. Do we have web developers willing to create such site?
3. Release files YUM/DNF/APT repositories,  ISO images hosting. After some point it should be ability to update NEXTSPACE with `yum` or 'apt' tools without manual install. Any ideas how we can bring this to life?

**Comment by trunkmaster** [comment]
NEWS: I've created `nextspace` branch in GNUstep repos for `libs-back` and `libs-gui` to hold some patches specific to NEXTSPACE or rejected by GNUstep or not commited by me to `master` branch for various reasons.
Quick instruction how to build GNUstep GUI `nextspace` branch for those who are not familiar with it:
```
$ git clone https://github.com/gnustep/libs-gui
$ cd libs-gui
$ git checkout nextspace
$ ./configure && make 
# su -
# make install
```
GNUstep Back process looks the same except the last 3 commands. It should look like this:
```
$ ./configure --enable-graphics=art --with-name=art && make 
$ su -
# make fonts=no install
```

**Comment by armm77** [comment]
hi @trunkmaster, thanks for the help in the construction of your project, I can suggest some small changes:
> 
> ```
> $ git clone https://github.com/gnustep/libs-base
> $ cd libs-base
> $ ./configure --disable-mixedabi && make 
> # su -
> # make install
> # ldconfig
> ```
> ```
> $ git clone https://github.com/gnustep/libs-gui
> $ cd libs-gui
> $ git checkout nextspace
> $ ./configure && make 
> # su -
> # make install
> # ldconfig
> ```
> 
> GNUstep Back process looks the same except the last 3 commands. It should look like this:
> 
> ```
> $ ./configure --enable-graphics=art --with-name=art && make 
> $ su -
> # make fonts=no install
> # ldconfig
> ```

**Comment by zeroheure** [comment]
>     1. This issue may be a place where I can put some hot news in Twitter like manner.  Sometimes I lack inspiration/time to write long posts on https://trunkmaster.github.io. Anyone interested can subscribe to this issue to be in touch with latest news and progress.

good idea

>     2. I guess it would be great some web page with project description and other materials related to the project. I'm do not have expertise in HTML, JavaScript and so on. Although GitHub has a place for it - GitHub Pages. Do we have web developers willing to create such site?

I can help

>     3. Release files YUM/DNF/APT repositories,  ISO images hosting. After some point it should be ability to update NEXTSPACE with `yum` or 'apt' tools without manual install. Any ideas how we can bring this to life?

SuSE tools ? "[Open Build Service (OBS)](https://openbuildservice.org/) is a generic system to build and distribute binary packages from sources in an automatic, consistent and reproducible way. You can release packages as well as updates, add-ons, appliances and entire distributions for a wide range of operating systems and hardware architectures. "

**Comment by trunkmaster** [comment]
@zeroheure, what do you need to create web page? I definitely need to check the OBS, thanks.

**Comment by zeroheure** [comment]
My first need is to install and setup my own build ! :-) It will help to take detailed screenshots and to comment about usability. You already wrote a lot of texts where I can pick elements for web pages.
I will work on a draft and show you.

**Comment by trunkmaster** [comment]
Quick update: I'm implementing a network preferences management. Last week I was trying to get familiar with [DBusKit](https://github.com/gnustep/libs-dbuskit). Good news: 
1. I've established communication to NetworkManager via D-Bus and got/set major preoperties/settings. The other option is to use GLib library (libnm) but it's more complicated - I'll stick to DBusKit.
2. DBusKit adoption opens ability to implement displaying of desktop notifications which are emitted by applications through D-Bus. Good. :)

P.S.: Although I've found a bug DBusKit and hope I'll send PR soon.

**Comment by zeroheure** [comment]
>     3. Release files YUM/DNF/APT repositories,  ISO images hosting. After some point it should be ability to update NEXTSPACE with `yum` or 'apt' tools without manual install. Any ideas how we can bring this to life?

See also [Multi Packager](https://rastersoft.com/programas/multipackager.html)

**Comment by alexmyczko** [comment]
i al interested in the apt part

**Comment by zeroheure** [comment]
Thanks for the upcoming release !

I wonder if you have a secret plan : I've read a recent paper from Red-Hat ; looks like NextSpace will be a perfect desktop (and foundation) for their projects ! :-)

**Comment by trunkmaster** [comment]
@zeroheure what paper? I'd like to read that paper too. 

I don't have any secret plan except for prepare to new intermediate release. Too much have happend in code (features, fixes, new GNUstep release with my numerous patches), more Linux distributions support is coming (CentOS 7, CentOS 8, Fedora 31).

**Comment by zeroheure** [comment]
@trunkmaster unfortunatly I don't remember. I started reading with a paper about Red Hat desktop plan and then browsed around. This isn't in my history but I've asked a friend if he could help to find it. 
When I've read that paper I immediaty thought that you are disruptive. NextSpace will come suddenly with advance "things" that we need.

**Comment by zeroheure** [comment]
@trunkmaster I started by reading Christian Schaller blog, then followed the links. The first and second post were :

1. https://blogs.gnome.org/uraeus/2020/04/28/fedora-workstation-swamp-draining-for-6-years/
1. https://blogs.gnome.org/uraeus/2020/04/24/a-bold-new-chapter-for-fedora-workstation/

**Comment by trunkmaster** [comment]
@zeroheure thank you for links. Quite interesting posts. RedHat/CentOS/Fedora are GNOME bounded distributions. And it's good because they are developing good underlying technologies (D-Bus, systemd, PulseAudio/PipeWire, NetworkManager etc). These technologies are good as a base for projects like NEXTSPACE. That's why I've chose CentOS for NEXTSPACE development.

**Comment by trunkmaster** [comment]
Guys, new blog pos is here https://trunkmaster.github.io.

**Comment by trunkmaster** [comment]
Hello everybody!

Please meet the [0.90 release](https://github.com/trunkmaster/nextspace/releases/tag/0.90).
Play, test and send me feedbacks (issues preferred).

Cheers

**Comment by zeroheure** [comment]
Hi Serji, 
Thankyou for the hardwork and the install scrip. What a pleasure to setup NextSpace so easily!
I have had some problems installing NextSpace for Centos 7 on real hardware. But I'm not sure if it comes from 0.90 or hardware bugs and unstable local network . I will restart a clean install tomorrow.
I also run it on a VM (kvm), still with Centos 7, and it run very well, except a little problem (see #276) easy to solve.

**Comment by zeroheure** [comment]
The consistency of NextSpace desktop is something other projects get only after a few years. At the beginnig, there is some "exotic" behaviour for one coming from Gnome or KDE but after few minutes, everything seems logical and **easy**. GNUStep as a desktop was never like that. You have build a fantastic desktop. Sincerily.

**Comment by trunkmaster** [comment]
@zeroheure thank you! I'm glad I've managed to translate original OPENTEP feel. :)

**Comment by tanshoku** [comment]
Gotta give my two bits here and also say that this project is amazing and I'm willing to help on what I can. I'm good with shell scripts so that's a thing I can help as needed, and I can also do a little bit of C. I was wondering if you guys want to have some sort of "community/developer hub" around discord, irc, or something like that, or if this issue thread here is enough for that =)

**Comment by trunkmaster** [comment]
@tanshoku thank you. I think you can help in Debian packaging (I put suggestions into issue #319).

As for chats, I'm afraid it will be to hard to gather online. I guess maillist/forum is preferred.

**Comment by zeroheure** [comment]
@trunkmaster I can buy a domain name for you and setup a mailing-list on a server.

**Comment by tanshoku** [comment]
Yes, I hadn't thought about that, but forums or a mailing list would be really way better than IM, since its easily accessible on the internet.
